x-pack/plugins/alerting/common/routes/rule/apis/bulk_delete/validation/v1.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
+import { BulkDeleteRulesRequestParams } from '../types/v1';
```

Comment:
might be clearer if we import `BulkDeleteRulesRequestParamsV1` from the `index` file

Reply:
2410993ae1d6047cbb272d1af947aa0e8b1acb85